views/components/ConfigurationsList/index.jsx

```diff
@@ -1,32 +1,22 @@
 import React from 'react'
-import GridList from '@material-ui/core/GridList'
+import { ImageList } from '@mui/material'
 import ConfigurationCard from '../ConfigurationCard'
 
-const ConfigurationsList = ({
-  classes,
-  configurations,
-  gridState,
-  ...rest
-}) => {
+const ConfigurationsList = ({ configurations, gridState, ...rest }) => {
   return (
     <>
-      <GridList
-        className={classes.gridList}
-        cols={gridState.columns}
-        cellHeight="auto"
-      >
+      <ImageList cols={gridState.columns} cellHeight="auto">
         {configurations.map((config) => {
           return (
             <ConfigurationCard
               key={config._id}
-              classes={classes}
               config={config}
               width={gridState.gridWidth}
```

Comment:
where does `gridState` come from?

Reply:
The gridstate comes from a hook. 
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)